src/ashley/machina_extensions/forum/views.py

```diff
@@ -256,6 +256,20 @@ class ForumLockCourseView(PermissionRequiredMixin, UpdateView):
     fields = ["lti_contexts"]
     template_name = "forum/forum_lock.html"
 
+    def get_list_forums(self):
```

Comment:
Since what you are getting is a list, I would call it:
```suggestion
    def get_forums_list(self):
```